portfolio/src/main/webapp/index.html

```diff
@@ -5,7 +5,8 @@
         <title>My Portfolio</title>
         
         <link rel="stylesheet" href="style.css">
-        <script src="script.js"></script>
+        <!-- <script src="script.js"></script> -->
```

Comment:
In the future, you don't need to leave things like this commented because you can see them anyway in revision history.